__tests__/attachment.spec.tsx

```diff
@@ -19,42 +17,56 @@ function buildSource(attachment?: string): string {
 `;
 }
 
-const inlineAttachment = "![image.png](attachment:spot-the-difference-2a.jpg)";
-
 const cellAttachments = {
-  "spot-the-difference-2a.jpg": {
-    "image/jpeg":
-      "iVBORw0KGgoAAAANSUhEUgAAAAUAAAAFCAYAAACNbyblAAAAHElEQVQI12P4",
-  },
-  "someotherimage.png": {
-    "image/png": "someothercontent",
-  },
+  "spot-the-difference-2a.jpg":
+    "data:image/jpeg;base64,iVBORw0KGgoAAAANSUhEUgAAAAUAAAAFCAYAAACNbyblAAAAHElEQVQI12P4",
+  "someotherimage.png": "data:image/png;base64,someothercontent",
 };
 
-const attchmnts: Attachments = toAttachments(cellAttachments);
+const inlineAttachment = "![image.png](attachment:spot-the-difference-2a.jpg)";
 
-//uses value 'mocked' defined in test-setup.js for return of URL.createObjectURL. Jest can't execute logic that uses this method.
 describe("attachment", () => {
-  test("toAttachments properly converts jupyter nb format cell attachments to markdown AST based attachments", () => {
-    expect(attchmnts["spot-the-difference-2a.jpg"]).toEqual("mocked");
-    expect(attchmnts["someotherimage.png"]).toEqual("mocked");
+  test("Attachment should be added to source if the source contains a valid jupyter attachment reference", () => {
+    const wrapped = mount(
+      <MarkdownRender
+        source={buildSource(inlineAttachment)}
+        attachments={cellAttachments}
+      ></MarkdownRender>
+    );
+    const imgProps = wrapped.find("img").props();
+    expect(imgProps).toHaveProperty(
+      "src",
+      "data:image/jpeg;base64,iVBORw0KGgoAAAANSUhEUgAAAAUAAAAFCAYAAACNbyblAAAAHElEQVQI12P4"
+    );
   });
 
-  test("Attachments should be added to source if there is matching reference", () => {
+  test("Source is unaffected when it contains jupyter attachment reference and cell attachments is not provided", () => {
+    const wrapped = mount(
+      <MarkdownRender source={buildSource(inlineAttachment)}></MarkdownRender>
+    );
+    const imgProps = wrapped.find("img").props();
+    expect(imgProps).toHaveProperty(
+      "src",
+      "attachment:spot-the-difference-2a.jpg"
```

Comment:
I'm curious what this is going to be like for users. When their image isn't available, it's going to have a 404 for this URL -- I wonder if we should include an `alt` tag or something on the element that says "cell attachment missing".

Reply:
I like the idea of using an alt tag.

Reply:
As a throwback, I'd love if we had 

![image](https://user-images.githubusercontent.com/836375/86153541-b44d4b00-bab6-11ea-87e3-8ba2bc781e10.png)